### ISSUE TITLE
Smokey Fails to Run on Windows

### ISSUE THREAD

**Issue by teward** [Charcoal-SE/SmokeDetector#555]
Excluding the test failures issue in #540 (and the now-merged fix in #544 that will fix the failure in the 'tests' sections which was necessary for these tests and running into this issue), we have a reproducible runtime failure in Windows environments.  That is, we get errors such as these repeatedly:

```
Git support not available in Windows.
teward/Aurora
https://metasmoke.erwaysoftware.com
Git support not available in Windows.
LoginError: failed to get `usr` cookie from Stack Exchange OpenID
2017-02-26 18:45:08.642000 UTC
  File "ws.py", line 97, in <module>
    GlobalVars.wrap.login(username, password)

  File "C:\Users\Administrator\Documents\SmokeDetector\ChatExchange\chatexchange\client.py", line 138, in login
    self._br.login_se_openid(email, password)

  File "C:\Users\Administrator\Documents\SmokeDetector\ChatExchange\chatexchange\browser.py", line 154, in login_se_openid
    "failed to get `usr` cookie from Stack Exchange OpenID")
```

I have gotten this error four separate times - in Windows 10, in Windows 7, in Windows Server 2012, and in Windows Server 2016 (which was on EC2 just to make sure it wasn't my environments).  In all four cases, this error has occurred, preventing Smokey from running.

This suggests to me that SmokeDetector is incapable of running on a Windows environment as a result of something in ChatExchange.  I'd like to propose allowing us to run the "tests" on Windows, but actually *exit* if someone tries to actually execute SmokeDetector on a Windows environment, given that I've had these ChatExchange-originating errors in four separate Windows test environments.

**Comment by AWegnerGitHub** [comment]
Chat exchange works fine on Windows. I am using it in another application.

The only thing I asked for a fix, because the only spot I am getting an
error, is the "sh" module. If I comment out the sh import, smokey will log
in and run just fine under my account. All I want is a way to eliminate the
git tests on a windows environment.

The discussions about production smokey run locations are way out of scope
in my issue. All I need/want is a development environment that will run the
tests without hacky workarounds like commenting out an import.

**Comment by teward** [comment]
@AWegnerGitHub We're beyond the scope of your issue since I already merged a fix for yours - that's not why this was referencing the other posts you made.  The scope of your issue was #540 which was fixed in #544 and is already merged.  We're on additional discussions taken place in CharcoalHQ about "Wanting to rule out Windows as a viable runtime target entirely."

It also seems the CE issues being observed here happen randomly or are environment specific, nevertheless clean Windows environments can't 'run' this currently.
